backend/dev.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mhart/alpine-node:10

```

Comment:
This creates another docker file that instead of using `npm run start` that uses .JS files, uses `npm run dev` that uses .TS files converting them on the go using a dedicated module